src/App.tsx

```diff
@@ -1,7 +1,33 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', (event: KeyboardEvent) => {
+      this.setState({ pressedKey: `The last pressed key is [${event.key}]` });
+    });
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', (event: KeyboardEvent) => {
+      this.setState({ pressedKey: `The last pressed key is [${event.key}]` });
+    });
```

Comment:
```suggestion
  componentDidMount() {
    document.addEventListener('keyup', someMethod);
  }

  componentWillUnmount() {
    document.removeEventListener('keyup', someMethod);
```
Create a method to handle it in order not to duplicate the same logic twice